### ISSUE TITLE
Ils not showing

### ISSUE THREAD

**Issue by DH174** [flybywiresim/a32nx#5202]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-25T14:50:34+00:00",
  "ref": "master",
  "sha": "1f9462ceda1a2c63c61437cd67ed60bc7b30cd2c",
  "actor": "tracernz",
  "event_name": "manual"
}
```

### Describe the bug

Hi. Don't know if maybe i am doing something wrong. I am using the experimental version of the A320. Did a clean install. Aircraft is flying the selected ils approach but the ils approach is not displayed on any of the displays. It grabs the g/s and does auto-landing perfectly but does not show which ils on which runway did it lock onto.

EDIT: also doesn't work on dev.

Any help?

### Expected behavior

ILS used to be displayed on the mdc. Expecting to see on which runway ILS has been locked onto.

### Steps to reproduce

Did a clean install after updating installer 

### Discord Username (optional)

DH174

**Comment by MisterChocker** [comment]
Please use the development or stable version. The experimental version is no longer being updated and therefore outdated.

**Comment by DH174** [comment]
I did a clean install of the development model and still the same result. The plane follow the flightpath and do the auto-landing perfectly. Its shows the magenta indicators of the g/s. It grabs the g/s and follow it. However there is no indication above the ARC which runway frequency it detected. I entered the ils frequency in the RAD NAV. When you switch the mode selector on the ARC to IS it picks up the frequency. When you switch back to ARC there is nothing.  The wind direction and speed indicator also started flashing. Sure I might be missing something?

**Comment by MisterChocker** [comment]
Can you please update the build info then?

**Comment by MisterChocker** [comment]
Do you have any other addons installer or in your community folder?

**Comment by DH174** [comment]
Fixed. Think one of my liveries was the culprit. Thanks

**Comment by DH174** [comment]
Nope not the livery. When i enter the flight plan via MSFS flight planner it shows. When I enter it myself in the FMS it does not. I have entered the ils approaches many times before with no problems.

**Comment by DH174** [comment]
{
  "built": "2021-06-25T14:50:34+00:00",
  "ref": "master",
  "sha": "1f9462ceda1a2c63c61437cd67ed60bc7b30cd2c",
  "actor": "tracernz",
  "event_name": "manual"
}

**Comment by MisterChocker** [comment]
> Do you have any other addons installer or in your community folder?

**Comment by DH174** [comment]
No. I cleaned out the community folder

**Comment by MisterChocker** [comment]
Can you provide a screenshot of the pfd and nd?

**Comment by DH174** [comment]
I am trying to recreate it to take a screenshot but as luck will have it everything works now. I will try and fly a complete flight and see if it does it again.

**Comment by DH174** [comment]
![20210626090755_1](https://user-images.githubusercontent.com/86413201/123505343-c5478280-d65e-11eb-916e-522dc2a63c5c.jpg)

Here you can see that the G/S indicators has been picked up and the ILS frequency is displayed bottom left in magenta but on the right display above the ARC there is no indication as to the approach that is being flown.

**Comment by DH174** [comment]
![20210625234708_1](https://user-images.githubusercontent.com/86413201/123505426-561e5e00-d65f-11eb-8ffa-9e29af419f74.jpg)

Here is a screenshot of another flight where it is indicated above the ARC

**Comment by tracernz** [comment]
Is it a custom scenery or something where you have issues? The localiser name (which is erroneously displayed at the top of the ND) comes from the navdata.

This is already fixed on the new ND, where it correctly displays the approach name in 9 characters.

**Comment by DH174** [comment]
No custom scenery. Cleared all addon scenery and mods.

**Comment by DH174** [comment]
Only thing I did was to update msfs2020 airac cycle via naviigraph to the latest airac cycle. Could this have any influence on what is displayed in the MD?

**Comment by kilocharlie1** [comment]
Do you have the LS button pressed?

**Comment by tracernz** [comment]
> to capture the localizer you have to manually input the localizer beacon frequency into the RAD NAV page on the FMS.

That is not true, unless the navdata for the airport is wrong.

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
